generator.yaml

```diff
@@ -135,3 +135,6 @@ ignore:
     - ModifyReplicationGroupInput.SecurityGroupIds
     - ModifyReplicationGroupInput.EngineVersion
     - CreateReplicationGroupInput.GlobalReplicationGroupId
```

Comment:
There are more fields for log delivery in the output than what are there in the input (Spec).
Those fields are ([api reference](https://awscli.amazonaws.com/v2/documentation/api/latest/reference/elasticache/create-replication-group.html)):
```
Status -> (string)
    Returns the log delivery configuration status. Values are one of enabling | disabling | modifying | active | error
Message -> (string)
    Returns an error message for the log delivery configuration.
```
Both of these fields provide useful information about actual status of this configuration.
As Spec values of saved K8s resource are not updated from the API output, the information contained in Status, and Message fields will not surface.

Thus, consider:
1) adding custom ko.Status fields to replication group:
```
LogDeliveryConfigurations:
    is_read_only: true
    from:
        operation: DescribeReplicationGroups
        path:LogDeliveryConfigurations
```
2) Consider setting resource sync conditions based on `LogDeliveryConfigurations.Status` value by checking it against value `active`
3) Consider setting condition message from `LogDeliveryConfigurations.Message` when `LogDeliveryConfigurations.Status` is `error`